src/components/sections/HowItWorks.tsx

```diff
@@ -1,55 +1,92 @@
-import { ArrowRight } from "lucide-react";
-
 const steps = [
-	{
-		number: "01",
-		title: "Register and Setup",
-		description: "Register and set up a customized gateway with your branding.",
-	},
-	{
-		number: "02",
-		title: "Integration",
-		description:
-			"Integrate PayStell with your store and set payment parameters.",
-	},
-	{
-		number: "03",
-		title: "Accept Payments",
-		description:
-			"Accept payments in local currency via Stellar-powered transactions.",
-	},
+  {
+    number: "01",
+    title: "Register & Setup",
+    description: "Get paid instantly, anywhere in the world",
+  },
+  {
+    number: "02",
+    title: "Share Link",
+    description: "Get paid instantly, anywhere in the world",
+    content: (
+      <div className="flex flex-col items-center justify-center w-full p-4 ">
+        <h4 className="text-sm font-semibold mb-2 text-left w-full">
+          Share your payment link
+        </h4>
+        <div className="border border-gray-100 rounded-lg shadow-lg w-full mb-4 p-4">
+          <p className="text-xs mb-2 text-left w-full">Share as you prefer</p>
+
+          <div className="w-full mb-2">
+            <input
+              type="text"
+              value="https://link.paystell.com/11aa22"
+              readOnly
+              className="w-full p-2 text-xs border rounded-lg"
+            />
+          </div>
+
+          <button className="bg-[rgb(1,158,255)] text-white text-sm font-semibold w-full p-1 rounded-md">
+            Copy Link
+          </button>
+        </div>
+        <div className="border border-gray-100 rounded-lg shadow-lg w-full mb-4 p-2">
+          <p className="text-xs mt-2 mb-2 text-left w-full ">
+            Charge for a payment button on your website
+          </p>
+
+          <div className="w-full mb-2">
+            <input
+              type="text"
+              value={`<script src="https://integrate.paystell.com/"`}
+              readOnly
+              className="w-full p-2 text-xs border rounded-lg"
+            />
+          </div>
+        </div>
+      </div>
+    ),
+  },
+  {
+    number: "03",
+    title: "Receive Payments",
+    description: "Get paid instantly, anywhere in the world",
+  },
 ];
 
 const HowItWorks = () => {
 	return (
-		<section id="how-it-works" className="py-24 bg-background">
-			<div className="container">
-				<div className="text-center max-w-3xl mx-auto">
+		<section id="how-it-works" className="py-24 bg-background" aria-labelledby="how-it-works-title">
+			<div className="container w-full max-w-full">
+				<div className="flex justify-center items-center mb-4">
+					<div className="px-3 py-1 text-sm font-semibold rounded-full bg-[rgb(229,247,255)] text-[rgb(1,158,255)] flex items-center gap-1" >
+						BENEFITS <img src="/files/star.png" alt="start in benefits" className="w-5 h-5"/>
+					</div> 
+				</div>	
```

Comment:
```suggestion
				<div className="flex justify-center items-center mb-4">
            <Badge className="w-fit bg-primary/10 px-3 rounded-xl hover:bg-primary/10">
              <h2 className="text-lg font-bold text-primary mr-3">HOW IT WORKS</h2>
            <Image src="/star.svg" width={28} height={28} alt="star" />
          </Badge>
				</div>	
```

Reply:
Hi @MPSxDev changes requested looks like this, could you please confirm? text should be HOW IT WORKS? and image position changed?

<img width="1419" alt="image" src="https://github.com/user-attachments/assets/bc0ba74a-23ca-4b34-a743-d8dca751fb27" />

Reply:
The image is in the source repo, it will be ok if I merge it, thank you great job

Reply:
You can commit the changes

Reply:
@MPSxDev I already committed the changes, let me know please if something else is needed